test/language/expressions/arrow-function/arrow/apply_2.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using apply - 'this' is lexically bound
+es6id: 14.2
+---*/
+
+var b = {
+    method: () => this
+ }

```

Comment:
This test is using the global object as the captured `this`, and the object literal `b` is superfluous to the test.  Two things here:

1. Was there any intention here to be validating that `b` is not captured as `this`?  If so that is unclear to me as a reader and should be clarified.

2. This test will pass 3 of the four 4 assertions below when performed with a function expression instead of an arrow function because the global object is the value bound to `this`.  Using a sentinel object instead of the global object for the captured `this` would ensure that all four assertions fail for non-arrow functions and thus ensure the asserts are actually verifying arrow function behavior.  E.g.

```javascript
var b = {}; // sentinel
var a = (function () { return () => this; }).call(b); // get arrow function object with this bound to `b`

assert.sameValue(a.apply(null), b, "..."); // etc.
```